### PR TITLE
Support for PHP 8

### PR DESCRIPTION
**Pull request by nebc-jasonfung**
- Upgrade to php 8.0 and its dependencies

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require" : {
-        "php": "^7.1",
+        "php": "^8.0",
```
Comment (hkdobrev) [inline composer.json:13]: Could you please change that to support both 7 & 8? E.g.:
```suggestion
        "php": "^7.3 || ^8",
```

We still use PHP 7.3 in some places and many people have not yet upgraded. If there is no code restriction to support both, I'd prefer support 7.x for some more time.

**Comment by hkdobrev** [comment]
@tbg-jasonfung Many thanks for the contribution!

We will also need to update `.travis.yml` and the PHP versions on which this is tested.
Could you look into that, please? Much appreciated!

**Comment by hkdobrev** [comment]
I guess we can drop 7.1 and 7.2, because of PHPUnit support and release as a new version instead of a new patch

**Comment by tbg-jasonfung** [comment]
> I guess we can drop 7.1 and 7.2, because of PHPUnit support and release as a new version instead of a new patch

@hkdobrev Ah yes, makes sense, changes made.

**Review by hkdobrev — APPROVED** [review]

**Comment by hkdobrev** [comment]
@tbg-jasonfung Thank you for your contribution! PHP 8.0 support has been released in 0.7.0!

(Releasing as a new minor bump under major version 0 as we're dropping support for previously supported PHP 7.1 and 7.2 versions.)

**Pull request merged**